src/Uno.UI.TestComparer/AzureDevopsDownloader.cs

```diff
@@ -52,32 +52,41 @@ public async Task DownloadArtifacts(string basePath, string project, string defi
 			{
 				var tempFile = Path.GetTempFileName();
 
-				Console.WriteLine($"Getting artifact for build {build.Id}");
-				using (var stream = await client.GetArtifactContentZipAsync(project, build.Id, artifactName))
+				var artifacts = await client.GetArtifactsAsync(project, build.Id);
+
+				if (artifacts.Any(a => a.Name == artifactName))
```

Comment:
alright sweet. So we are now removing the assumption that there always will be an artifact. 

LGTM..

Everything below this line is whitespace noise.